requirements.txt

```diff
@@ -1,26 +1,54 @@
 annotated-types==0.5.0
+asttokens==2.4.0
```

Comment:
Are all of these new additions really necessary? I tried it with the old requirements.txt and it worked nicely.

Reply:
Hey, I don't think they are necessary, I will just remove them and check once.